src/charm.py

```diff
@@ -510,6 +506,35 @@ def _config_force_https(self) -> None:
         )
         process.wait_output()
 
+    def _on_anonymize_user_action(self, event: ActionEvent) -> None:
+        """Anonymize data from a user.
+
+        Args:
+            event: Event triggering the anonymize_admin_user action.
+        """
+        username = event.params["username"]
```

Comment:
Would be useful to check if the user is already anonymized?

Is it possible to do this even with administrator users?

Reply:
If it has, the anonymization will be applied again, having no effect whatsoever.

Yes, it is possible to anonymize admin usres.